content/pages/whoweare.html

```diff
@@ -121,23 +121,6 @@ <h3>Austin Macdonald</h3>
 		</div>
 	 </div>
 
-  <div class="row">
-    <h3 id="vadim_melnik_">Vadim Melnik</h3>
-    <figure><img src="/theme/img/team/vadim.jpg" class="pull-left"/></figure>
-    <div class="col-md-4 col-sm-4 position">Software Engineer &amp; Consultant</div>
-    <div class="col-md-5 col-sm-5 contact">
-      <a href="mailto:vmelnik@docsultant.com?Subject=Hello Vadim M. @CON" class="email">E-Mail</a>
-      <a href="https://www.docsultant.com" class="home">Home Page</a>
-      <a href="https://twitter.com/vmdocua" class="twitter">Twitter</a>
-      <a href="https://github.com/vmdocua" class="github">GitHub</a>
-      <a href="https://www.linkedin.com/in/vadim-melnik" class="linkedin">LinkedIn</a>
-    </div>
-    <div class="col-md-9 col-sm-9 description">
-      External consultant working on ReproStim, ReproMon, con/noisseur, events collection/timing
-      projects and cosmic harmony.
-    </div>
-  </div>
-
```

Comment:
you must not delete unrelated to your changes content.  please rewrite your commit to avoid this change

Reply:
done